### PR TITLE
Move docs out of repo

### PR DESCRIPTION
**Pull request by gbraad**
The user documentation has moved to a new repository at https://github.com/crc-org/docs

Related to: https://github.com/crc-org/crc-internal/issues/91

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: go get && go build ./... && go test ./... && make
+    command: go run .
+
+
```
Comment (praveenkumar) [inline .gitpod.yml]: Do we really need this file to be part of this commit, this is more like your local dev setup.

Comment (gbraad) [reply .gitpod.yml]: This is not local dev setup, but was not suppoed to be checked in.

**Comment by gbraad** [comment]
Documentation is already publishing to https://crc.dev/docs, so this functionality is not needed anymore; please merge soon to prevent conflicting index by search engines

**Comment by gbraad** [comment]
👍, also updated the blog to point to the "Getting Started Guide"

**Pull request merged**